apps/system/js/card.js

```diff
@@ -90,30 +96,32 @@
    * @memberOf Card.prototype
    */
   Card.prototype.template = function() {
-    return Sanitizer.escapeHTML `<div class="titles">
-     <h1 id="${this.titleId}" dir="auto" class="title">${this.title}</h1>
-     <p class="subtitle">
-      <span class="subtitle-url">${this.subTitle}</span>
-     </p>
-    </div>
-
-    <div class="screenshotView bb-button" data-l10n-id="openCard"
-      role="link"></div>
-    <div class="privateOverlay"></div>
-    <div class="appIconView" style="background-image:${this.iconValue}"></div>
-
-    <footer class="card-tray">
-     <button class="appIcon" data-l10n-id="openCard"
-       data-button-action="select" aria-hidden="true"></button>
-     <menu class="buttonbar">
-       <button class="close-button bb-button" data-l10n-id="closeCard"
-         data-button-action="close" role="button"
-         style="visibility: ${this.closeButtonVisibility}"></button>
-      <button class="favorite-button bb-button"
-        data-button-action="favorite" role="button"
-        style="visibility: ${this.favoriteButtonVisibility}"></button>
-     </menu>
-    </footer>`;
+    return Sanitizer.escapeHTML `<section class="content">
```

Comment:
I don't think we need the extra `section` dom container anymore, since we're not doing vertical APZ.
I was tempted to leave that as a follow up, but the history of the task manger shows us that we should never let unused cruft settle in. So I'd like to remove it for v1 of the patch.
(might help with some test failure too since we'll be bringing back the old dom)